tests/test_dag/test_dag.py

```diff
@@ -1,9 +1,24 @@
+from unittest import mock
 from airflow.models import DagBag
 
+from test_dag import task_1, task_2
 
+
+@mock.patch("springer.dag_pull_ftp.SpringerRepository", mock.MagicMock())
+@mock.patch("springer.dag_pull_ftp.SpringerSFTPService", mock.MagicMock())
 def test_dag_loaded():
     dagbag = DagBag(dag_folder="dags/", include_examples=False)
     dag = dagbag.get_dag(dag_id="test_dag")
-    assert dagbag.import_errors == {}
+    assert dagbag.import_errors.get("dags/test_dag.py") == None
     assert dag is not None
     assert len(dag.tasks) == 2
+
+
+def test_task1():
+    value = task_1()
+    assert value == "first_task"
+
+
+def test_task2():
+    value = task_2()
+    assert value == "second_task"
```

Comment:
we don't need those any more, you can remove them with the test_dag file as well

Reply:
Done